hrc_discrim_learning/ObjPercept[] objs
